lbz/dev/test.py

```diff
@@ -11,25 +15,68 @@ class Client:
     Client created for testing purposes.
     """
 
-    def __init__(self, resource):
+    def __init__(self, resource: Resource):
         self.resource = resource
 
-    def post(self, resource: str, params=None, query_params=None, body=None, headers=None):
+    def post(
+        self,
+        resource: str,
```

Comment:
The same name: `resource` but differently used within one class 😕

Reply:
renamed to path, but there is slight chance it will break stuff in other places if someone used it as kwargs.

Reply:
Yes, that's true, but for me, it's a really small change (easy to apply) and the benefits of cleaner code are huge 😌  Thanks!